src/Config/EventConfigurator.php

```diff
@@ -47,4 +47,13 @@ public function registerListeners(array $eventsListeners): void
             }
         }
     }
+
+    private function isCallable($definition): bool
+    {
+        if (is_callable($definition)) {
+            return true;
+        }
+
+        return count($definition) === 2 && in_array($definition[1], get_class_methods($definition[0]) ?? [], true);
```

Comment:
Why do you not check that $definition is an array?

Reply:
Done.

Reply:
https://github.com/yiisoft/yii-web/commit/61be1117b9d4a2dfa6b793b1af297896198248fd#r39247634